time-app.html

```diff
@@ -98,6 +98,9 @@ <h3 class="white f3">Become a BETA tester!</h3>
               <input id="app_email" name="app_email" class="br2 ba dwyl-b--dark-gray pa2 w-100 border-box" type="email" value="" required placeholder="tim@cern.com" />
```

Comment:
Does this actually need an `id`? 

Could it just have `name="email"`, making it easier to DRY up the handler?

Reply:
I think it's only defined that way because there was a page where there were two forms ...
can be changed in next iteration. 👍